broker-daemon/broker-rpc/wallet-service/commit-balance.js

```diff
@@ -71,9 +71,19 @@ async function commitBalance ({ params, relayer, logger, engines }, { EmptyRespo
     throw new PublicError(`Maxium balance of ${MAX_CHANNEL_BALANCE} exceeded for committing to the relayer. Please try again.`)
   }
 
+  const {maxBalance: maxOutboundBalance} = await engine.getMaxChannel()
+  const {maxBalance: maxInboundBalance} = await inverseEngine.getMaxChannel({outbound: false})
+  const convertedBalance = convertBalance(balance, symbol, inverseSymbol)
+  if (maxOutboundBalance || maxInboundBalance) {
+    if ((maxOutboundBalance && Big(maxOutboundBalance).lte(balance)) || (maxInboundBalance && Big(maxInboundBalance).lte(convertedBalance))) {
```

Comment:
Do we want `lte` here? I figured we would just want to throw if the balance is less than the amount